spec/spree_shared_spec.rb

```diff
@@ -21,6 +21,7 @@
     end
     
     Apartment::Tenant.switch!
+    Spree::Preferences::Store.instance.persistence = true
```

Comment:
Did you find out why this is required? The way it works is super weird

Reply:
I didn't nope, I would like to learn more about module including/extending/prepending in general. I feel like we'll need that for our reports gem